pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -233,11 +232,6 @@ func (l LibvirtWrapper) StartLibvirt(stopChan chan struct{}) {
 			exitChan := make(chan struct{})
 			args := []string{"-f", "/var/run/libvirt/libvirtd.conf"}
 			cmd := exec.Command("/usr/sbin/libvirtd", args...)
-			if l.user != 0 {
```

Comment:
@rmohr This should not be removed.

Reply:
Reverted. Maybe I should give libvirtd the capability to avoid that we have to do this switch. Will check that in a follow-up.

Thanks.